tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -110,16 +112,21 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
                             Ok((d.to_string() + input, Consumed::Empty(input)))
                         })));
 
-    let query_parser = try(string(QUERY_COMMAND)).or(try(string(ALT_QUERY_COMMAND)))
+    let edn_arg_parser = || spaces()
```

Comment:
really? I could've sworn I totally rewrote that. 

Looking at it, I rewrote it blow here but didn't delete the old one. I think things might've gone wiggy in the rebasing.